app/src/main/java/fr/free/nrw/commons/theme/BaseActivity.java

```diff
@@ -27,6 +28,8 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         wasPreviouslyDarkTheme = systemThemeUtils.isDeviceInNightMode();
         setTheme(wasPreviouslyDarkTheme ? R.style.DarkAppTheme : R.style.LightAppTheme);
+        float fontScale = android.provider.Settings.System.getFloat(getBaseContext().getContentResolver(),android.provider.Settings.System.FONT_SCALE, 1f);
```

Comment:
The line is larger than the allowed codestyle. So while passing values in the method, putting a new line after comma would be better

Reply:
Larger indeed, well observed! Personally I would break after the first parenthesis, but one new line per argument is fine too.